packages/knip/src/plugins/xo/types.ts

```diff
@@ -0,0 +1,15 @@
+import type { ESLintConfig } from '../eslint/types.js';
+
+export type XOConfig = ESLintConfig & {
+  envs?: string[] | undefined;
+  globals?: string[] | undefined;
+  parser?: string | undefined;
+  plugins?: string[];
```

Comment:
Not sure what's already in `ESLintConfig` or should be over there?

Reply:
I've deduped them, thx